src/pages/Home/components/About.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
```

Comment:
The "About" and "How it works" sections have the same design, so you can use one component called something like `Info` or `InfoSection` or so. This should have margins or padding in place so you can add as many sections as you like without touching the styles again.

Then you can wrap both of these sections in another component which has margins to fit in between the heatmap and the footer.

I hope this was understandable!?

Reply:
so I should remove howitwork.js and about.js and use one component Info.js ?